contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelItem.vue

```diff
@@ -432,4 +432,9 @@
     width: 100%;
   }
 
+  .channel-cards {
+    display: inline-block;
```

Comment:
What was your reasoning for `inline-block` here? This is troubling me because it seems that the way it's displayed seems to align better with `block` styling.

Reply:
My reasoning for using ‘inline-block’ was because it allows for a width to be set on an element and would keep the padding and margin, but since the elements are on top of one another and we want it to take up the whole width, it does make sense to use ‘block’. Also yes, I would like more information on how to avoid the overflow out of the viewport issue.

Reply:
So regarding the viewport overflow, the first thing to note is that the `ChannelItem` component is used in 2 places, `ChannelList.vue` and `CatalogList.vue`. It seems we only have this issue on the `CatalogList` view. I also note that `ChannelItem` has a `fullWidth` prop. It seems that internally the component uses `fullWidth` for a few things (probably could be refactored in the future as it seems rather complicated).

In `ChannelList.vue`, `ChannelItem` is inside a block element, so naturally this should expand to full width of that element. In `CatalogList` though, this is in a `display: flex` element, which I believe actually causes this issue because normally you would expect a block element to expand to the full width unless otherwise defined. Enabling `fullWidth` in `CatalogList` seems to cause more issues, so that seems like a non-option here. Given this is implemented in a flex container in `CatalogList`, it seems that the appropriate solution would be to apply `flex-grow: 1` on its implementation in `CatalogList`.

Reply:
I did apply the `flex-grow: 1` on the channel cards implementation in `CatalogList`, but the overflow issue continued. Since the viewport overflow occurred when I set the `min-width` to 100%, I found the easiest solution was to remove this and set the width of the cards in the `ChannelList.vue` to 100%  and it solved the issue of the cards without descriptions not filling the complete horizontal space and the viewport overflow.

Reply:
Yeah the 100% change should no longer be needed with `flex-grow: 1`